### DISCUSSION TITLE
Seurat Cluster Tree

### DISCUSSION THREAD

**Discussion by danaschwalbe** [satijalab/seurat#4599] [Q&A] [answered]
Hi!

I am working on producing a cluster tree for the data I am analyzing. The cluster trees I produce have unusually large blue boxes at the nodes of the tree. My colleagues have run the same code and do not have this issue. 
Does anyone know of arguments I can add to BuildClusterTree() or PlotClusterTree() that would fix this issue?

Thank you!

**Comment by timoast** [comment, marked as answer]
You probably need to increase the size of the plotting device
